### PR TITLE
Microverse JavaScript Capstone Submission

### PR DESCRIPTION
**Pull request by joshuaoivie**
Dear Code Reviewer,

This pull request contains the following changes:

Add Home Page
List Movies
List Movie Likes
Add Movie Like Functionality
Add Details Modal
Add Comment List
Add New Comment Functionality.
Thank you for your anticipated favourable response.

Here's a video recording highlighting some things I enjoyed about this project: [Video Recording](https://www.loom.com/share/d115e463bbe24d0b9e1fa8a762ee8052)

Warm Regards,
Joshua Ivie.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  </header>
+  <main id="main"></main>
+  <footer>
+    Built with <span class="love">❤️</span> by <a href="https://github.com/joshuaivie" target="_blank"
+      rel="noopener">Joshua Ivie</a> & <a href="https://github.com/JuanLPalacios" target="_blank" rel="noopener">Juan
+      Luis Palacios</a>.
+  </footer>
+</body>
+
+</html>
```
Comment (micheaol) [inline src/index.html]: Please ensure that the `Home page footer` is similar to the given [mockup](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md) according to the project requirement.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Capstone 2</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+</head>
+
+<body>
+  <header>
+    <h1 class="logo">Movie <span class="slender">Somollier</span></h1>
+  </header>
```
Comment (micheaol) [inline src/index.html]: Please ensure that the `Home page header` and navigation are similar to the given [mockup](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md) according to the project requirement.

File: src/tests/home.test.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  { id: 2, name: 'a', image: { medium: '' } },
+  { id: 3, name: 'a', image: { medium: '' } },
+]);
+// eslint-disable-next-line no-unused-vars
+const getDetails = jest.fn((id) => getList()
+  .then((list) => list.filter((item) => item.id === id)[0]));
+
+describe('', () => {
+  it('should add a count to the list', async () => {
+    const list = await seriesCount(getList());
```
Comment (micheaol) [inline src/tests/home.test.js:14]: Please ensure that `ALL` the members of the team appear in the video(presentation) and talk about the project according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes

- _Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tests/home.test.js (hunk 3 above)
Comment (joshuaoivie) [reply src/tests/home.test.js:14]: I'm working alone at the moment.

Comment (micheaol) [reply src/tests/home.test.js:14]: @joshuaivie ,

Please is there any particular reason why you are working alone at the moment?

Comment (joshuaoivie) [reply src/tests/home.test.js:14]: I was unpaired from my partner by student success.

Comment (micheaol) [reply src/tests/home.test.js:14]: @joshuaivie ,

Oh so sorry about that. Kindly make other changes and re-submit for review from your dashboard.

File: src/index.html (hunk 2 above)
Comment (joshuaoivie) [reply src/index.html]: The header has been updated

File: src/index.html (hunk 1 above)
Comment (joshuaoivie) [reply src/index.html]: The footer has been updated

**Comment by joshuaoivie** [comment]
All requested changes have been effected.

**Review by andersonlebon — APPROVED** [review]
### Hi @joshuaivie 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by joshuaoivie** [comment]
Thanks, @micheaol and @andersonlebon!

**Pull request merged**